src/templates/actions/thunks.ts

```diff
@@ -289,3 +291,22 @@ export const fetchAndSetStacks = (orgID: string) => async (
     console.error(error)
   }
 }
+
+export const fetchAndSetReadme = (name: string, directory: string) => async (
+  dispatch: Dispatch<Action>
+): Promise<void> => {
+  try {
+    const response = await fetchReadMe(directory)
+    dispatch(setTemplateReadMe(name, response))
+  } catch (error) {
+    reportError(error, {
+      name: 'The community template fetch github readme failed',
```

Comment:
```suggestion
      name: `The community template github readme fetch failed for ${name}`,
```
Just a little bit of extra context to help us if there's a problem